### ISSUE TITLE
Incompatible shapes of X(data) and Y(label)

### ISSUE THREAD

**Issue by xinge333** [kuleshov/audio-super-res#18]
Hi Volodymyr ,

Thanks for sharing your nice work. I have some problems when I making the preparation of data, I set parameters as like this --stride=4096, --dimension=8192. But after I finish the data preparation, I found that the data(lr_patches)'s dimension and the label(hr_patches)'s dimension don't match. They are (1600, 4096, 1) and (1600, 8192, 1), as you can see, the second dimension is not same. 
This problem results at the problem of calculating loss when I train the model, the bug is shown as:

  File "/home/disk1/xyzhou/workspace/audio-super-res-py2/audio-super-res/src/models/model.py", line 72, in create_train_op
    self.loss = self.create_objective(X, Y, opt_params)
  File "/home/disk1/xyzhou/workspace/audio-super-res-py2/audio-super-res/src/models/model.py", line 102, in create_objective
    sqrt_l2_loss = tf.sqrt(tf.reduce_mean((P-Y)**2 + 1e-6, axis=[1,2]))

InvalidArgumentError (see above for traceback): Incompatible shapes: [64,4096,1] vs. [64,8192,1];

Sorry to bother you. Please let me know if you have any suggestions. Thank you.

Best regards,
xinyong

**Comment by shairoz** [comment]
same issue, any update on how to resolve?

**Comment by xinge333** [comment]
We must modify the code customly.

**Comment by JahnKhan** [comment]
have a look at the paper [image super-resolution with deep neuronal networks](https://arxiv.org/abs/1501.00092) they described it. The low-resolution picture is upscaled in the preprocessing step. 
In this project you can use the --interpolate argument for doing so. After that, the low-resolution inpout should have the same dimension.

**Comment by thomasr96** [comment]
If you look at the Kuleshov paper, they say in the methods section, "To ensure source/target series are of the same length, the source input is pre-processed with cubic upscaling"

**Comment by Sawyerb** [comment]
@thomasr96 is correct. If you're using the process and parameters from the README, then you shouldn't run into any dimensionality issues when training,
